corfudb-tools/src/main/java/org/corfudb/browser/CorfuBrowser.java

```diff
@@ -0,0 +1,73 @@
+package org.corfudb.browser;
+
+import com.google.common.reflect.TypeToken;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.collections.CorfuDynamicKey;
+import org.corfudb.runtime.collections.CorfuDynamicRecord;
+import org.corfudb.runtime.view.TableRegistry;
+import org.corfudb.util.serializer.DynamicProtobufSerializer;
+import org.corfudb.util.serializer.ISerializer;
+import org.corfudb.util.serializer.Serializers;
+
+@Slf4j
+public class CorfuBrowser {
+    private final CorfuRuntime runtime;
+    private final String singleNodeEndpoint;
+    private static final int SERIALIZER_TYPE = 25;
+
+    public CorfuBrowser(String host, int port, String keyStore,
+        String ksPasswordFile, String trustStore, String tsPasswordFile,
+        boolean tlsEnabled) {
+        runtime = CorfuRuntime.fromParameters(
+            CorfuRuntime.CorfuRuntimeParameters.builder().useFastLoader(false)
+        .cacheDisabled(true)
+        .tlsEnabled(tlsEnabled)
+        .keyStore(keyStore)
+        .ksPasswordFile(ksPasswordFile)
+        .trustStore(trustStore)
+        .tsPasswordFile(tsPasswordFile).build());
+        singleNodeEndpoint = String.format("%s:%d", host, port);
+        connectRuntime(singleNodeEndpoint);
+    }
+
+    private void connectRuntime(String endpoint) {
+        runtime.parseConfigurationString(endpoint);
+        runtime.connect();
+        log.info("Successfully connected to {}", endpoint);
+    }
+
+    public CorfuTable getTable(String namespace, String tableName) {
+        ISerializer dynamicProtobufSerializer =
+            new DynamicProtobufSerializer((byte)SERIALIZER_TYPE, runtime);
+        Serializers.registerSerializer(dynamicProtobufSerializer);
+
+        CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> corfuTable =
+            runtime.getObjectsView().build()
+            .setTypeToken(new TypeToken<CorfuTable<CorfuDynamicKey, CorfuDynamicRecord>>() {
+            })
+            .setStreamName(
+                TableRegistry.getFullyQualifiedTableName(namespace, tableName))
+            .setSerializer(dynamicProtobufSerializer)
+            .open();
+        return corfuTable;
+    }
+
+    @SuppressWarnings("checkstyle:printLine")
```

Comment:
Since this is only for testing. Remember to remove this before checking in.

Reply:
done